src/routes/cache-hooks/cache-hooks.service.ts

```diff
@@ -44,6 +45,7 @@ export class CacheHooksService {
   async onEvent(
```

Comment:
We are missing the type on `cache-hooks.controller.ts`. We should consider adding a union type given the amount of types involved here 😅

Reply:
Makes total sense to me, I'll add the union type in a separate PR if you agree!

(I've added the new type to `CacheHooksController` in [27f4dee](https://github.com/safe-global/safe-client-gateway/pull/981/commits/27f4dee496e4a740c986c8a6b0887501a324e6ff), thanks! 🙂)